app/views/activities/_filters.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with url: project_activities_path(current_project), method: :get, remote: true, class: 'd-flex flex-xl-row flex-column gap-3', data: { behavior: 'activity-filters' } do |f| %>
+  <div class="d-flex flex-column flex-sm-row w-100 gap-3">
+    <div class="w-100">
+      <%= f.label :user, class: 'form-label' %>
+      <%= f.select :user, options_from_collection_for_select(current_project.authors.enabled, :id, :name, params[:user]), { include_blank: 'All users' }, { class: 'form-select' } %>
+    </div>
+    <div class="w-100">
+      <%= f.label :type, class: 'form-label' %>
+      <%= f.select :type, options_from_collection_for_select(current_project.activities.pluck(:trackable_type).uniq, :to_s, :demodulize, params[:type] ), { include_blank: 'All types', selected: params[:type] }, { class: 'form-select' } %>
+    </div>
+  </div>
+
+  <div class="w-100">
+    <span class="form-label mb-2 d-block">Date range</span>
+    <div class="d-flex flex-column flex-sm-row gap-3 w-100">
+      <div class="input-group">
+        <%= f.date_field :period_start, max: params[:period_end], placeholder: 'From date', value: params[:period_start], class: 'form-control form-control-sm', onkeydown: "return false" %>
```

Comment:
let's move `onkeydown` to the related js file and [avoid inline js](https://github.com/dradis/dradis-ce/wiki/Style-Guide:-JavaScript#code)